### PR TITLE
Add not support for holdings

### PR DESCRIPTION
**Pull request by matthew55**
As it is right now, it is not possible to use `not` for holdings. This PR makes it easier when dealing with issues such as #275 to run `!rsa holdings all not chase`.

### CODE REVIEW

**Comment by matthew55** [comment]
Also, unrelated but later on in the file `autoRSA.py` there is the code sample:
```python
        if not orderObj.get_holdings():
            printAndDiscord("All transactions complete in all brokers", loop)
```
which prevents a message from showing that holdings command has finished to completion. Can I remove the check preventing a holdings completion message?

**Comment by NelsonDane** [comment]
> Also, unrelated but later on in the file `autoRSA.py` there is the code sample:
> 
> ```python
>         if not orderObj.get_holdings():
>             printAndDiscord("All transactions complete in all brokers", loop)
> ```
> 
> which prevents a message from showing that holdings command has finished to completion. Can I remove the check preventing a holdings completion message?

You're good to remove that, I would just change the wording to something like "All commands complete in all brokers" instead of the word "transactions". It was there since very often holdings was often 1 account but it makes sense to print a completed.

**Comment by matthew55** [comment]
Should be ready for a review and merge.

File: autoRSA.py [hunk 1]
```diff
@@ -182,6 +181,13 @@ def argParser(args: list) -> stockOrder:
         else:
             for broker in args[1].split(","):
                 orderObj.set_brokers(nicknames(broker))
+
+        # If next argument is not, set not broker
+        if len(args) > 3 and args[2] == "not":
+            for broker in args[3].split(","):
+                if nicknames(broker) in SUPPORTED_BROKERS:
+                    orderObj.set_notbrokers(nicknames(broker))
+
```
Comment (NelsonDane) [inline autoRSA.py:190]: Looks good, can you remove the newlines on 184 and 190 to match the rest of the function?

**Review by NelsonDane — CHANGES_REQUESTED** [review]

**Review by NelsonDane — APPROVED** [review]
Awesome, thanks so much!

**Pull request merged**